lib/rules/class-methods-use-this.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * @fileoverview Rule to enforce that all class methods use 'this'.
+ * @author Patrick Williams
```

Comment:
Awesome. There is a copyright in the tests though, which should be removed.
Thanks for clarifying!

On Aug 13, 2016 11:03 AM, "Ilya Volodin" <notifications@github.com> wrote:

> In lib/rules/class-methods-use-this.js
> <https://github.com/eslint/eslint/pull/6881#discussion_r74688636>:
>
> > @@ -0,0 +1,78 @@
> > +/**
> > + * @fileoverview Rule to enforce that all class methods use 'this'.
> > + * @author Patrick Williams
>
> No copyright. We don't need copyrights inside rules, we have one copyright
> at the root, that covers the whole repository.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/eslint/eslint/pull/6881/files/72790ca4a28556d53d48477775615cdcb189c263#r74688636>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AARWejOkuNuQegDwJcvmHc0vP8Nh1vfxks5qferKgaJpZM4JiKR9>
> .
>